app/assets/javascripts/utilities/buildArticleHTML.js.erb

```diff
@@ -20,34 +20,45 @@ function buildArticleHTML(article) {
+      tagString = tagString + '<a href="/t/'+t+'" class="crayons-tag"><span class="crayons-tag__prefix">#</span>'+t+'</a>\n'
       });
     }
-    var commentsCountHTML = ""
-    if ((article.comments_count || '0') > 0 && article.class_name != "User") {
-      commentsCountHTML = '<div class="article-engagement-count comments-count"><a href="'+article.path+'#comments"><img src="<%= asset_path("comments-bubble.png") %>" alt="chat" /><span class="engagement-count-number">'+(article.comments_count || '0')+'</span></a></div>'
+
+    var commentsDisplay = ''
+    var commentsCount = '0'
+
+    if ((article.comments_count || '0') > 0) {
+      var commentsCount = (article.comments_count || '0')
+    }
+    if (article.class_name != "User") {
+      var commentsDisplay = '<a href="'+article.path+'#comments" class="crayons-btn crayons-btn--s crayons-btn--ghost crayons-btn--icon-left"><svg class="crayons-icon" width="24" height="24" xmlns="http://www.w3.org/2000/svg"><path d="M10.5 5h3a6 6 0 110 12v2.625c-3.75-1.5-9-3.75-9-8.625a6 6 0 016-6zM12 15.5h1.5a4.501 4.501 0 001.722-8.657A4.5 4.5 0 0013.5 6.5h-3A4.5 4.5 0 006 11c0 2.707 1.846 4.475 6 6.36V15.5z"/></svg>'+commentsCount+'<span class="hidden s:inline">&nbsp;comments</span></a>'
     }
+
     var flareTag = ""
     if (container){
       var currentTag = JSON.parse(container.dataset.params).tag
     }
     if (article.flare_tag && currentTag != article.flare_tag.name) {
-      flareTag = "<span class='tag-identifier' style='background:"+article.flare_tag.bg_color_hex+";color:"+article.flare_tag.text_color_hex+"'>#"+article.flare_tag.name+"</span>"
+      flareTag = "<span class='crayons-story__flare-tag' style='background:"+article.flare_tag.bg_color_hex+";color:"+article.flare_tag.text_color_hex+"'>#"+article.flare_tag.name+"</span>"
     }
     if (article.class_name == "PodcastEpisode"){
-      flareTag = "<span class='tag-identifier'>podcast</span>"
+      flareTag = "<span class='crayons-story__flare-tag'>podcast</span>"
     }
     if (article.class_name == "Comment"){
-      flareTag = "<span class='tag-identifier'>comment</span>"
+      flareTag = "<span class='crayons-story__flare-tag'>comment</span>"
     }
     if (article.class_name == "User"){
-      flareTag = "<span class='tag-identifier' style='background:#5874d9;color:white;'>person</span>"
+      flareTag = "<span class='crayons-story__flare-tag' style='background:#5874d9;color:white;'>person</span>"
     }
+
     var rc = article.positive_reactions_count
-    var reactionsCountHTML = ''
-    if ((rc || '0') > 0 && article.class_name != "User") {
-      var reactionsCountHTML = '<div class="article-engagement-count reactions-count"><a href="'+article.path+'"><img src="<%= asset_path("reactions-stack.png") %>" alt="heart" /><span id="engagement-count-number-'+article.id+'" class="engagement-count-number">'+(rc || '0')+'</span></a></div>'
+    var reactionsCount = (rc || '0')
+    var reactionsDisplay = ''
```

Comment:
missing semicolons

Reply:
Ditto :) Basically this file contain elements that are in injected into the feed dom after the initially loaded (in logged out state), and upon scrolling for more results (both logged in and logged out states).

Our mixed-use of Preact for the feed in logged in versus logged out and loading results from the database versus ElasticSearch has resulted in this code which we hope to refactor in the future.